html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <div>
+        <div class="Email">
+          <label for="Email">Email</label>
+          <br/>
+          <input type="text" name="email" id="email" required>
+        </div>
+        <div>
+          <div>
+            <label for="Question">Why is coding so cool?</label>
+            <br/>
+            <select name="dropdown">
```

Comment:
This is not properly configured. The `for` attribute` needs to match the `id` attribute on the form control element this is labeling.